src/style/blocks/card.css

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product-img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+
+  display: block;
+}
+
+.card__product-name {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  
+  margin-bottom: 16px;
+
+  color: #616070;
+}
+
+.card__product-rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  margin-bottom: 24px;
+}
+
+.card__review {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  margin-bottom: 16px;
+}
+
+.card__price-paragraph {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #616070;
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__product-buy {
+  height: 40px;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+
+  background: #00acdc;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+
+  color: #fff;
+  transition: 0.6s;
+}
+
+.card__product-buy:hover {
+  color: #00acdc;
+
+  background: #fff;
+  transition: 0.3s;
```

Comment:
we need to change only color appearance by ":hover" 
"transition" must be in main class for bottom  "buy"